integration_tests/run_pyspark_from_build.sh

```diff
@@ -66,6 +66,12 @@ else
         echo "xdist not installed cannot run tests in parallel"
     fi
 
+    ACCEPTANCE_PARAM=""
+    if [[ "${TEST_ACCEPTANCE}" != "" ]];
```

Comment:
So this variable simply has to exist and be non-empty?  Behavior might be surprising if it was set to `0`, `NO`, etc.  Should this be checking for `true` or `True` instead of any non-empty val?